src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/PersonsTest.kt

```diff
@@ -199,6 +201,9 @@ class PersonsTest: AbstractTest() {
             )
 
             val expectedWeeks = ceil(daysBetweenMonth / 7.toDouble()).toInt()
+            println(expectedWeeks)
```

Comment:
We generally try to avoid leaving logs, is there a specific reason for these print lines?